schema-templates/basic-auth.sql

```diff
@@ -0,0 +1,72 @@
+begin;
+
+create schema if not exists basic_auth;
+set search_path to basic_auth;
```

Comment:
Maybe no search path? I've found it is really hard for new developers to understand. Clarity is important if this is meant to be a learning resource.